cardsWorthTen.js

```diff
@@ -21,7 +72,25 @@ const cards = [
  * @param {array} cards
  * @return {string} displayVal
  */
-const cardsWorthTen = cards => {};
+const cardsWorthTen = cards => {
+
+  let filteredCards = cards.filter((card) => card.val === 10);
+  let filteredVal = [];
+
+  for (let i = 0; i < filteredCards.length; i++) {
+
+    filteredVal = console.log(`${filteredCards[i].displayVal}`);
```

Comment:
Ah . . . . The key here is what we want the function to return. Remember that console.log() is just a side effect . . . it simply executes the writing of values to the console. It does not return values themselves. So what happens here is that the program executes the log on each loop, but `filteredVal` is always `undefined`.

So going back to the hint on line 69: `returns a string of the card display values where the value is equal to 10`

so this `filteredVal that you retrun on line 86 ought to be a string that you build up from the filtered card results.

Reply:
I get that I have to turn this into a string but I can't come up with a successful approach.  I tried creating another array called stringVal and then used both join ( stringVal =  filteredVal.join(', ');) and stringify (JSON.stringify(filteredVal)) to try to get them to link together into that array.  I keep getting an error.  Everything I read about join and stringify talks about stringing/joining properties in a single object.  I don't see anything about joining values in a property across multiple objects.